### ISSUE TITLE
Issue (Pages item below of the table)

### ISSUE THREAD

**Issue by johnkalangha** [osTicket/osTicket#5484]
![image](https://user-images.githubusercontent.com/30627667/80597707-caa93d00-8a5a-11ea-8072-de14eca12573.png)

Ticket Filter Paged
![image](https://user-images.githubusercontent.com/30627667/80597778-e6144800-8a5a-11ea-9aa2-dc04fa97c1fc.png)
i only have one page but it show 10pages here!

Specific Ticket Audit Log

i only have 5 items, but it show 10 pages. if i click page 2, all tabs will load and it will go back to  open ticket

**Comment by JediKev** [comment]
@johnkalangha 

>i only have one page but it show 10pages here!

Yes, pagination has been like this since at least v1.12.5 due to this commit:
- https://github.com/osTicket/osTicket/commit/18c93119c6d84877c27eb46aae1b7fda631b95b0

We are going to rework pagination eventually.

> i only have 5 items, but it show 10 pages. if i click page 2, all tabs will load and it will go back to open ticket

Yes, we believe this is an issue with the Audit plugin code so stay tuned for an update soon.

Cheers.

**Comment by JediKev** [comment]
@johnkalangha 

Audit Plugin Pagination is addressed with:
- https://github.com/osTicket/osTicket-plugins/pull/180

New plugin builds coming by the end of the day today or tomorrow so stay tuned.

Cheers.
